roxctl/central/debug/download_diagnostics_test.go

```diff
@@ -0,0 +1,105 @@
+package debug
+
+import (
+	"archive/zip"
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stackrox/rox/pkg/utils"
+	"github.com/stackrox/rox/roxctl/common/environment"
+	"github.com/stackrox/rox/roxctl/common/flags"
+	io2 "github.com/stackrox/rox/roxctl/common/io"
+	"github.com/stackrox/rox/roxctl/common/printer"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDownloadDiagnosticsTimeoutReached(t *testing.T) {
+	shutdownServer := make(chan struct{}, 1)
+	server := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
+		<-shutdownServer
+	}))
+	defer server.Close()
+	defer close(shutdownServer)
+
+	_, stdErr, err := executeDiagnosticsCommand(t, server.URL, 10*time.Millisecond, "")
+
+	require.Error(t, err)
+	assert.True(t, isTimeoutError(err))
+	assert.Contains(t, stdErr.String(), "Timeout has been reached while creating diagnostic bundle")
```

Comment:
nit: use a const instead of re-using this one within both `TestDownloadDiagnosticsTimeoutReached` & `TestDownloadDiagnosticsServerError`.